src/pip/_internal/commands/install.py

```diff
@@ -306,8 +306,8 @@ def run(self, options: Values, args: List[str]) -> int:
             options.target_dir = os.path.abspath(options.target_dir)
             if (
                 # fmt: off
-                os.path.exists(options.target_dir) and
-                not os.path.isdir(options.target_dir)
+                os.path.exists(options.target_dir)
+                and not os.path.isdir(options.target_dir)
```

Comment:
This bothers me because it's in a `# fmt: off` block. Does ruff not support that directive?

Reply:
Ruff is more limited in where it supports supression statements: https://github.com/astral-sh/ruff/discussions/6338

I suspect this is a location where it does not support them, ruff has been considering adding a warning for this case: https://github.com/astral-sh/ruff/issues/6611